OracleLinuxDevelopers/oraclelinux7/nodejs/14-oracledb/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+FROM oraclelinux:7-slim
+
+RUN yum -y install oracle-nodejs-release-el7 oracle-instantclient-release-el7 && \
+    yum -y install nodejs node-oracledb-node14 && \
+    rm -rf /var/cache/yum/*
+
```

Comment:
Add `ENV NODE_PATH=/usr/lib/node_modules` like the other Node.js oracledb Dockerfiles. (Untested; I'll test later).

Reply:
Good catch.